modules/ROOT/pages/authentication-authorization/manage-users.adoc

```diff
@@ -579,12 +794,16 @@ With `ENCRYPTED`, the password string is expected to be in the format of `<encry
 *** `0` is the first version and refers to the `SHA-256` cryptographic hash function with iterations `1`.
 *** `1` is the second version and refers to the `SHA-256` cryptographic hash function with iterations `1024`.
 * If the optional `SET PASSWORD CHANGE [NOT] REQUIRED` is omitted, the default is `CHANGE REQUIRED`.
-The `SET PASSWORD` part is only optional if it directly follows the `SET PASSWORD` clause.
+* The `SET PASSWORD` prefix of the `CHANGE [NOT] REQUIRED` clause is only optional if it directly follows the `SET PASSWORD 'password'` clause and is not part of a `SET AUTH` clause.
 * The default for `SET STATUS` is `ACTIVE`.
 * `SET HOME DATABASE` can be used to configure a home database for a user.
 A home database will be resolved if it is either pointing to a database or a database alias.
 If no home database is set, the DBMS default database is used as the home database for the user.
-* The `SET PASSWORD CHANGE [NOT] REQUIRED`, `SET STATUS`, and `SET HOME DATABASE` clauses can be applied in any order.
+[role=label--new-5.24]
```

Comment:
I don't think these labels in the list render?

Reply:
hmm, no, they don't do they. Any idea @renetapopova ? Thanks.

Reply:
I'll take a look when I start reviewing the PR. Is it ready for review on my end?